app/services/session.js

```diff
@@ -8,7 +8,7 @@ export default ESASession.extend({
   setCurrentUser: function() {
     if (this.get('isAuthenticated')) {
       let id = this.get('session.authenticated.user_id');
-      this.get('store').findRecord('current-user', id).then((user) => {
+      this.get('store').findRecord('user', id).then((user) => {
         this.set('currentUser', user);
```

Comment:
We should really consider getting rid of the `session.currentUser` and relying instead on `currentUser.user`.